CONTRIBUTING.md

```diff
@@ -87,19 +87,12 @@ A particular issue with testing this package is reproducibility. For the most pa
 This issue surfaces itself when testing the print messages of the `pool` object which displays treatment estimates which are thus not identical when run on different machines. To address this issue pre-made `pool` objects have been generated and stored in `R/sysdata.rda` (which itself is generated by `data-raw/create_print_test_data.R`). The generated print messages are compared to expected values which are stored in `tests/testthat/_snaps/` (which themselves are automatically created by `testthat::expect_snapshot()`)
 
 
-## glmmTMB
+## Fitting MMRM's
 
-The `glmmTMB` package is responsible for fitting MMRM models in `rbmi` and as such is a critical dependency. As `glmmTMB` is a maturing package under active development there have been occasions where issues have cropped up; if you do experience any issues with `glmmTMB` whilst developing `rbmi` please do raise a ticket on their repository [here](https://github.com/glmmTMB/glmmTMB).
+This package currently uses the mmrm package to fit MMRM models. This package is still fairly
+new but has so far proven to be very stable, fast and reliable. If you do spot any issues
+with the MMRM package please do raise them in the corresponding GitHub Repository - [link](https://github.com/openpharma/mmrm/issues)
 
-The most common issue is a warning that `glmmTMB` has been compiled against a different version of `TMB` or `Matrix` compared to what is  installed locally. The only main to this is to manually recompile the package yourself by running:
-
-```
-install.packages(c("TMB", "glmmTMB"), type = "source")
-``` 
-
-Note that you will need to have [rtools](https://cran.r-project.org/bin/windows/Rtools/rtools42/rtools.html) installed if you are on a Windows machine or [Xcode](https://apps.apple.com/us/app/xcode/id497799835?mt=12) if you are running macOS (or somehow else have access to a C/C++ compiler).
```

Comment:
I think this would be good to keep, also applies here

Reply:
Ah, yes you still use `TMB` so yer this issue has not fully gone away. Good spot will add back in.